src/composables/pollAccountInfo.ts

```diff
@@ -0,0 +1,29 @@
+import { IDefaultComposableOptions } from '@/types';
+import { createPollingBasedOnMountedComponents } from '@/composables/composablesHelpers';
+import { useFungibleTokens } from './fungibleTokens';
+
+let isInitialized = false;
+
+export function usePollAccountInfo({ store }: IDefaultComposableOptions) {
+  const {
+    loadTokenBalances,
+    loadAvailableTokens,
+  } = useFungibleTokens({ store });
+
+  const defaultNamesWatcher = createPollingBasedOnMountedComponents(10000);
+  const availableTokensWatcher = createPollingBasedOnMountedComponents(60000);
+  const tokenBalancesWatcher = createPollingBasedOnMountedComponents(10000);
+
+  function startPolling() {
+    defaultNamesWatcher(() => store.dispatch('names/setDefaults'));
+    availableTokensWatcher(() => loadAvailableTokens());
```

Comment:
I think within this PR we can consider not loading available tokens on initialization.
- all the tokens are only needed in receive modal token list
- currently token balances endpoint contains all the needed info to show them, without loading all available tokens
- we would need to add new method load all the needed info for a particular token in several places

Reply:
https://aeternity-blockchain.atlassian.net/jira/software/c/projects/SW/boards/2?selectedIssue=SW-1330

Reply:
i prepared new task for it

Reply:
I don't think creating separate task for this suggestion is a good thing. Actually @pstachel , @CedrikNikita and me made the same comment and I'm almost 100% sure we all can agree that this composable file is a wrong implementation of `createPollingBasedOnMountedComponents`.